daemon/src/identify.rs

```diff
@@ -0,0 +1,173 @@
+use crate::Environment;
+use std::collections::HashSet;
+
+pub mod dialer;
+pub mod listener;
+pub mod protocol;
+
+pub const PROTOCOL: &str = "/itchysats/id/1.0.0";
```

Comment:
not relevant to this PR, but a meta-comment : it's a bit sad that we can't reconcile between `PROTOCOL ` and `PROTOCOL_NAME`  :/ mixing 2 is unnecessary 😢

Reply:
https://github.com/itchysats/itchysats/pull/2583 :)